kalite/distributed/tests/browser_tests/distributed.py

```diff
@@ -289,32 +289,6 @@ def test_exercise_mastery(self):
         self.assertEqual(elog.attempts_before_completion, self.nanswers, "Student should have %s attempts for completion." % self.nanswers)
 
 
-@unittest.skipIf("medium" in settings.TESTS_TO_SKIP, "Skipping medium-length test")
-class LoadExerciseTest(BrowserActionMixins, KALiteBrowserTestCase):
-    """Tests if the exercise is loaded without any JS error.
-
-    The test is run over all urls and check for any JS error.
-    """
-    student_username = 'test_student'
-    student_password =  'socrates'
-
-    def setUp(self):
-        super(LoadExerciseTest, self).setUp()
-        self.student = self.create_student()
-        self.browser_login_student(self.student_username, self.student_password)
-
-    def test_get_exercise_load_status(self):
-        for path in get_exercise_paths():
-            logging.debug("Testing path : " + path)
-            self.browser.get(self.live_server_url + path)
-            error_list = self.browser.execute_script("return window.js_errors;")
-            if error_list:
-                logging.error("Found JS error(s) while loading path: " + path)
-                for e in error_list:
-                    logging.error(e)
-            self.assertFalse(error_list)
-
-
```

Comment:
Mostly because it is now testing code that is entirely Javascript. We should be doing this in a Javascript framework. And it usually gets skipped anyway.